shared/rakefiles/xk_util.rake

```diff
@@ -220,4 +220,35 @@ task :revoke_org_admin => [@gcp_creds_file, :configure_extra_tf_vars] do
   end
 end
 
+# This task restores a list of images in snapshots. 
+task :restore_snapshot_from_image_file, [:snapshot_files] => [@gcp_creds_file, :configure_extra_tf_vars] do |taskname, args|
+  require 'csv'
+
+  pv_zones = {}
+  CSV.parse(%x{
+              #{@exekube_cmd} kubectl get pv -o json | jq -r '.items[] | \"\\(.spec.claimRef.name),\\(.metadata.labels.\"failure-domain.beta.kubernetes.io/zone\")\"'
+           }.chomp).each do |line|
+             pv_zones[line[0]] = line[1]
+           end
+
+  snapshot_files = args[:snapshot_files].split ' '
+
+  snapshot_files.each do |snapshot_file|
```

Comment:
I guess this is just for debugging? Should probably have some text explaining this output.

Reply:
No, it is not for debugging, I added some comments in this commit https://github.com/gpii-ops/gpii-infra/pull/432/commits/f19194f2a1e1cc6ec51cdd12aa9eac14bb7ff4ae
I use the snapshot_files array to iterate the restoration of each pv.